eZ/Bundle/EzPublishCoreBundle/EventListener/CacheViewResponseListener.php

```diff
@@ -66,7 +66,14 @@ public function configureCache(FilterResponseEvent $event)
         }
 
         $response->setPublic();
-        if ($this->enableTtlCache) {
+
+        if (($maxAge = $view->getMaxAge()) !== null) {
+            $response->setMaxAge($maxAge);
+        }
+
+        if (($sharedMaxAge = $view->getSharedMaxAge()) !== null) {
+            $response->setSharedMaxAge($sharedMaxAge);
+        } elseif ($this->enableTtlCache) {
```

Comment:
For me, MaxAge / SharedMaxAge should be taken into account **only** if TTL cache is activated application wise (so only if `enabledTtlCache` is `true`).
If it is enabled, then one can selectively deactivate the cache via `CachableView::setCacheEnabled()`.